artifactory/services/utils/resultutils.go

```diff
@@ -34,15 +34,6 @@ type OperationSummary struct {
 	TotalFailed            int
 }
 
-// Represents a file transfer from SourcePath to TargetPath.
-// Each of the paths can be on the local machine (full or relative) or in Artifactory (full URL).
-// File's Sha256 calculated by Artifactory during upload. we read sha256 from the HTTP's response body.
```

Comment:
File's Sha256 calculated... --> The file's Sha256 is calculated...
...during upload.... --> ..during the upload....
we read sha256 from... --> We read the sha256 from...